docs/reference/esql/functions/kibana/definition/match_operator.json

```diff
@@ -0,0 +1,31 @@
+{
+  "comment" : "This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.",
+  "type" : "operator",
+  "name" : "match_operator",
+  "description" : "Performs a match query on the specified field. Returns true if the provided query matches the row.",
+  "signatures" : [
+    {
+      "params" : [
+        {
+          "name" : "field",
+          "type" : "keyword",
```

Comment:
You're right - I added the missing field types in 68907ff92de181ffe91c853fe33ebe7c309566b4.

> needs additional tests on text type for the operator

I think `text` type is already covered in CSV tests? Are there other tests you would expect to check this?

Reply:
I mean to add `text` type in the `MatchOperatorTests` so that the `text` type can be included in the docs automatically, which you already did in the latest commit.

Reply:
Just for completeness, in `docs/reference/esql/functions/kibana/definition/match.json`, we have 4 combinations:

keyword, keyword
keyword, text
text, keyword
text, text

`docs/reference/esql/functions/kibana/definition/match_operator.json` has 2 combinations:
keyword, keyword
text, text

Adding two additional tests for the other two combinations in `MatchOperatorTests` should be able to make the types of `:` the same as `match` function.

Reply:
Got it - adding them in cdf896ce90e74c90860675dd77538fa432470059